presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/QueryBuilder.java

```diff
@@ -285,6 +282,33 @@ private String toPredicate(String columnName, String operator, Object value, Jdb
         return quote(columnName) + " " + operator + " ?";
     }
 
+    private String getGroupBy(Optional<List<List<JdbcColumnHandle>>> groupingSets)
```

Comment:
it would be nice to have unit tests for this class that would test SQL string generation.

Reply:
We do not have any such tests yet. For example, we have `TestJdbcQueryBuilder` which is an integration test against H2.

Reply:
added